web/client/epics/__tests__/catalog-test.js

```diff
@@ -36,38 +36,5 @@ describe('catalog Epics', () => {
         });
 
     });
-    it('autoSearchEpic', (done) => {
```

Comment:
Why this has been removed?

Reply:
it was dispatching only a thunk that is tested in the actions/__tests__/catalog-test.js
one way to test it here is to transform the thunk into an epic, if you think is worth the shot, i'll do it